entrypoint.sh

```diff
@@ -233,9 +233,7 @@ main() {
 
   if [ "$JSON_OUTPUT" = "true" ] && [ "$LOG4J_CONFIG_FILE" = "log4j2.xml" ]; then
     echo "JSON_OUTPUT==true && LOG4J_CONFIG_FILE==log4j2.xml, so updating application.properties and log4j2.xml to output JSON to console"
-    sed -i '/<!-- p=priority c=category d=datetime t=thread m=message n=newline -->/d' $LOG4J_CONFIG_FILE
-    sed -i 's/<PatternLayout.*\/>/<JSONLayout compact="true" eventEol="true" properties="true" stacktraceAsString="true" \/>/' $LOG4J_CONFIG_FILE
-    sed -i 's/^logging.pattern.console/# logging.pattern.console/' config/application.properties
+    cp $LOG4J_CONFIG_FILE "$LABKEY_HOME/classes/configs/01.log4j2.xml"
```

Comment:
this path does not exist in a running container. I think you mean either `$LABKEY_HOME/config` or `$LABKEY_HOME/labkeywebapp/WEB-INF/classes` ?

Reply:
I was guessing at this one a bit. The expected URI is something along the lines of `classpath:configs/<order number>.log4j2.xml` So i'm not exactly sure what that would look like from the script side. The search string used is: `**/configs/*.log4j2.xml` the ** operator should glob folder path, and find it anywhere on the classpath under a `configs` dir.